packages/expo-image-picker/ios/UIImage+fixOrientation.swift

```diff
@@ -0,0 +1,80 @@
+// Copyright 2022-present 650 Industries. All rights reserved.
+
+import UIKit
+
+extension UIImage {
+  func fixOrientation() -> UIImage {
```

Comment:
There're far easier ways available in the Internet for fixing orientation (for sure shorter at least). I guess I might take a look at them and evaluate their correctness.